packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2172,6 +2172,17 @@ export class StripeHelper {
       throw error;
     }
 
+    // We make an exception here for customers because we need to get the
+    // subscriptions for the customer.  The Stripe API stopped including
+    // subscriptions for a customer in version 2020-08-27; this ensures
+    // backwards compatibility for our code that's relying on that behavior.
+    if (resourceType === CUSTOMER_RESOURCE) {
+      // @ts-ignore
+      return this.stripe[CUSTOMER_RESOURCE].retrieve(resource, [
+        SUBSCRIPTIONS_RESOURCE,
```

Comment:
This isn't the correct syntax for subscription expansion, but I think this should call the existing helper for loading a customer that's here because calling code here is quite likely to pass this object to our other functions which are faster if they preload the same things we frequently access on a customer object which our customer function loads.

Reply:
>This isn't the correct syntax for subscription expansion

D'oh!  Thanks for catching that.

> but I think this should call the existing helper for loading a customer 

By this you mean `StripeHelper.customer`?  I did look at that, and it does more than just getting the customer (in addition to a different interface requiring different arguments).  For example, code that calls `expandResource` to get a customer would explicitly check for `customer.deleted` on the return value, where as `fetchCustomer` (what `StripeHelper` ultimately calls to get a customer) handles `customer.deleted` and returns undefined.

For purpose of the expiring CC reminder bug, I could call `StripeHelper.customer` in `StripeHelper.extractSourceDetailsForEmail` and avoid this altogether.  What do you think?

Reply:
That's a good point, I think calling that in the other method instead of using expand resource on customers is a good approach. Might be useful to see if any other caller on `expandResource` uses customer, and if not, just remove it as a valid expand option in favor of always using the other method.

Reply:
> Might be useful to see if any other caller on expandResource uses customer, and if not, just remove it as a valid expand option in favor of always using the other method.

I already did, and there are a few.  None of them rely on the 'subscriptions' property though, so we've got that going for us at least.

Reply:
>  I think calling that in the other method instead of using expand resource on customers is a good approach. 

Looks like this isn't possible because of a chicken and egg problem. 😢 `Stripe.customer` requires a uid and an email addresses, but we won't have those until we get the customer.